### ISSUE TITLE
Changing Placeholder text in Azure web app

### ISSUE THREAD

**Issue by Farshad6788** [microsoft/sample-app-aoai-chatGPT#869]
Hi there,
I have customized the interface of my Azure web app, and now I'm trying to change the placeholder text. I followed the steps outlined in this [link](https://github.com/microsoft/sample-app-aoai-chatGPT/tree/main#common-customization-scenarios-eg-updating-the-default-chat-logo-and-headers) to customize my website app. I used environmental variables to achieve the customization, but I'm not sure which environmental variable to use to change the placeholder text. 
<img width="960" alt="S1" src="https://github.com/microsoft/sample-app-aoai-chatGPT/assets/148047508/9e941521-26b6-458d-9edd-f32bccfdfd31">
In other words, I am looking for a specific tag to add it as an Environmental variable to my app setting and change the text of placeholder, similar to what I did for UI_TITLE , UI_CHAT_TITLE and ... .
As you see in the attached image, I only added app settings based on the previous link (which I had mentioned above) and they were changed according to the defined values.
<img width="917" alt="Screenshot 2024-05-22 154339" src="https://github.com/microsoft/sample-app-aoai-chatGPT/assets/148047508/1b4cb944-f677-4e9d-97e4-436ded445fa9">
Can someone advise on how to do this?
Thank you.

**Comment by AlexKaSi** [comment]
You have do modify the Answer.tsx in frontend\src\components\Answer then build and redeploy it.

**Comment by Farshad6788** [comment]
@AlexKaSi 
Thanks for your response. What about with environmental variables? Isn't it possible by adding an environmental variable in app setting according to the second attached image?

**Comment by abhahn** [comment]
Hi @Farshad6788 , I would probably consider this to be an enhancement rather than a bug, since we don't currently have this capability implemented (and therefore can't be broken).  :)

I'll change the label to "enhancement" and add it to our backlog as another customization we can add.

**Comment by Farshad6788** [comment]
@abhahn 
Great. Thanks ;)

**Comment by AlexKaSi** [comment]
@abhahn maybe you could also add the "AI-generated generated content may be incorrect" & "Generating..." text in Answer.tsx as a label- this would make all relevant text-snippets easily modifiable
